src/arch/LowLevelWindow/LowLevelWindow_SDL.cpp

```diff
@@ -13,9 +16,9 @@ using namespace RageDisplay_Legacy_Helpers;
 
 #include <stack>
 #include <math.h>	// ceil()
-#include <SDL2/SDL.h>
-#include <SDL2/SDL_opengl.h>
-#include <SDL2/SDL_syswm.h>
+#include <SDL.h>
+#include <SDL_opengl.h>
+#include <SDL_syswm.h>
```

Comment:
There cannot be any assumptions on there being an SDL2 directory. `SDL2_INCLUDE_DIR` should handle that for us.